fc_util/src/bit_helper.rs

```diff
@@ -0,0 +1,186 @@
+/// Structure representing a range of bits in a number
```

Comment:
If we are not going to use this in other crates as well, we should have this as a module in cpuid. The plan for cpuid is to enhance it with other features that are more generic and can be used by other projects as well. This is why it's probably best not to have dependencies on Firecracker-specific crates in cpuid.